src/pkg/selectors/selectors_test.go

```diff
@@ -239,3 +240,126 @@ func (suite *SelectorSuite) TestSplitByResourceOnwer() {
 		})
 	}
 }
+
+// TestPathCategoriesAll includes all the included selectors do not return `Unknown` Category
+func (suite *SelectorSuite) TestPathCategoriesAll() {
+	users := []string{tester.LoadTestM365UserID(suite.T())}
+
+	table := []struct {
+		name        string
+		getSelector func(t *testing.T) *Selector
+		isErr       assert.ErrorAssertionFunc
+	}{
+		{
+			name:  "empty",
+			isErr: assert.Error,
+			getSelector: func(t *testing.T) *Selector {
+				return &Selector{}
+			},
+		},
+		{
+			name:  "Mail_B",
+			isErr: assert.NoError,
+			getSelector: func(t *testing.T) *Selector {
+				sel := NewExchangeBackup(users)
+				sel.Include(sel.MailFolders([]string{"MailFolder"}, PrefixMatch()))
```

Comment:
I'd suggest expanding this test to include a specific item in addition to the folder type.  There should probably be a test for filters as well, but this is enough coverage for now.
```suggestion
				sel.Include(
				    sel.MailFolders([]string{"MailFolder"}, PrefixMatch()),
				    sel.Mail([]string{"MailFolder2"}, []string{"Mail"}),
				)
```

Reply:
There are specific filter tests to ensure that abstraction was working. The gotcha that I found while looking into #2112 was that a `sel.Include(<topology>` was returning a category, `unknown` which causes an operation to fail. Will include a few others as random testing.

Reply:
Resolved within [fa9b7ad](https://github.com/alcionai/corso/pull/2111/commits/fa9b7ad72227a7b60a2fe1532d5112eb3a430654)